yvalenta/Example-socket/simple-client.rb

```diff
@@ -0,0 +1,12 @@
+require 'socket'      # Sockets are in standard library
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.